src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -79,47 +142,47 @@ protected String defaultToDashesIfBlank(String value, String propertyName) {
         return StringUtils.defaultIfBlank(value, collectorHelper.getDashOnlyPropertyValue(propertyName));
     }
 
-    public List<OriginEntryFull> buildOriginEntries() {
-        List<OriginEntryFull> originEntries = new ArrayList<>();
+    /**
+     * Generates OriginEntryFull BOs from the SAE lines and passes them to the given Consumer.

```

Comment:
Ok, I'll work on refactoring this into multiple methods for clarity and less commenting.